src/workers/validators.ts

```diff
@@ -211,19 +176,25 @@ export const fetchValidatorsInfo = async () => {
           addBidDelegatorSave(_delegator);
           totalDelegated += Number(ethers.utils.formatUnits(delegator.staked_amount, 9));
         });
+
         const totalBid = selfStake + totalDelegated;
         const selfStakePercentage = (selfStake / totalBid) * 100;
+        const totalValidatorRewards: number = (await getBidRewards(bid.public_key))[0]?.totalRewards || 0;
+        const totalDelegatorRewards: number = (await getBidDelegatorRewards(bid.public_key))[0]?.totalDelegatorRewards || 0;
 
         bidValidators.push({
           publicKey: bid.public_key,
           numOfDelegators: bid?.bid?.delegators?.length,
           delegationRate: bid.bid?.delegation_rate,
+          performance: eraValidatorsInfo.length / 360,
```

Comment:
The validator performance calculation is incorrect - refer to the previous implementation and make it a bid by bid case not all bids per era.